nodes/standalone/src/command.rs

```diff
@@ -16,21 +16,11 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-use crate::{
-	benchmarking::{inherent_benchmark_data, RemarkBuilder, TransferKeepAliveBuilder},
-	chain_spec,
-	cli::{Cli, Subcommand},
-	service,
-	service::ExecutorDispatch,
-};
```

Comment:
Because they do not show up anymore in the source code.

Reply:
I left only `use crate::cli::{Cli, Subcommand};`

Reply:
Also some of them were feature-gated, and for that I would rather use the full path than feature-gate `use`s.

Reply:
Which ones are feature-gated? Sorry, I don't see it.

I am asking because in `command.rs`, it uses `load_spec`, which was previously directly imported and is now updated to `crate::service::load_spec`.

Reply:
`ExecutorDispatch` is used with `try-runtime`. `benchmarking*` is not used anymore.

I re-imported `new_partial` and `load_spec` in https://github.com/KILTprotocol/kilt-node/pull/643/commits/5b18286e45c68b7c79f58524e1c6b265d4e130a6.